src/hotspot/share/opto/parse2.cpp

```diff
@@ -1371,10 +1371,27 @@ inline int Parse::repush_if_args() {
   return bc_depth;
 }
 
+// Used by StressUnstableIfTraps
+static volatile int _trap_stress_counter = 0;
```

Comment:
Please, check that all accesses to it use ExternalAddress (external_word_type relocation).